homekit/aio/controller/ip/pairing.py

```diff
@@ -272,21 +273,21 @@ def event_received(self, event):
 
         data = {'characteristics': data}
 
-        response = await self.connection.put_json('/characteristics', data)
-
-        char_set = set(characteristics)
         tmp = {}
 
+        try:
+            response = await self.connection.put_json('/characteristics', data)
+        except AccessoryDisconnectedError:
+            return {}
+
         if response:
             for row in response:
                 id_tuple = (row['aid'], row['iid'])
                 tmp[id_tuple] = {
                     'status': row['status'],
                     'description': HapStatusCodes[row['status']],
                 }
-                char_set.discard(id_tuple)
 
```

Comment:
can you remove this line as well? Then flake8 should be quiet again ;)

Reply:
Should be done (flake8 being weird on this machine)